CRM/Contact/Import/Form/MapField.php

```diff
@@ -396,12 +400,16 @@ public function buildQuickForm() {
     $formName = 'document.forms.' . $this->_name;
     //used to warn for mismatch column count or mismatch mapping
     CRM_Core_Session::singleton()->setStatus(NULL);
+    $processor = new  CRM_Import_ImportProcessor();
+    $processor->setMappingID($savedMappingID);
```

Comment:
Not actually using mappingId & metadata yet but it seemed like the 'right sized bite' to include these sets in this pr